updated BatchRunner

tpike3 opened the pull request:
Program throwing error when passing in agent reporters. Error at line 185
line 185: agent_record[vars] = reporter(agent)    *error* could not read string
updated line 185 to
line 185:  agent[vars]=getattr(agent, reporter)

program ran without error, produced appropriate results from datacollector/batchrunner

jackiekazil commented:
@tpike3 where you able to get passed the Travis build fail or do you need an extra pair of eyes on this?

tpike3 commented:
Ran the nosetests - issue was in the test_bathrunner did not pass in a string. I updated agent_reporters in test_batchrunner to fix errors

Also cleaned up some minor issues showing up on flake8

jackiekazil commented:
This all looks good to me! Ty for the fix!
